docs/source/releases/v1_12_0a0.rst

```diff
@@ -0,0 +1,33 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 1.12.0a0 (2024-02-12)
```

Comment:
Should this be 1.12.2a0 ?  I think it should match whatever is in geoips repo update_this_release_note file:
https://github.com/NRLMMD-GEOIPS/geoips/blob/main/update_this_release_note

Reply:
Past Mindy says that it should be 1.12.2a0, based on the GitHub Actions she set up:
https://github.com/NRLMMD-GEOIPS/geoips_clavrx/actions/runs/8694314032/job/23842934175?pr=34

Reply:
I updated these.